ipa-core/src/report/hybrid.rs

```diff
@@ -682,8 +688,8 @@ pub type PrfHybridReport<BK, V> = IndistinguishableHybridReport<BK, V, u64>;
 #[derive(Clone, Debug, Eq, PartialEq, Default)]
 pub struct IndistinguishableHybridReport<BK, V, MK = Replicated<BA64>>
 where
-    BK: SharedValue,
-    V: SharedValue,
+    BK: SharedValue + BooleanArray,
+    V: SharedValue + BooleanArray,
```

Comment:
```suggestion
    BK: BooleanArray,
    V: BooleanArray,
```

`BooleanArray` requires `SharedValue`, so I would favor conciseness.